app/routes/bookmarks/index.tsx

```diff
@@ -224,7 +230,10 @@ export function BookmarkEntryComponent({
           onClick={() => {}}
         />
       </div>
-      {open && (
+      <CollapseTransition
+        show={open}
+        className="duration-300 h-0 overflow-hidden"
+      >
```

Comment:
todo: actually we might not won't collapse as it would momentarily renders `MiniPlayer` right after next `bookmarkEntry` is picked up during practice loop.

Reply:
For now we can do remount hacking https://github.com/hi-ogawa/ytsub-v3/pull/209/commits/629cc4460d18d502e77f811fb6e39bfd7426f68e